AssetRipper.Core/Project/Collections/AssetsExportCollection.cs

```diff
@@ -91,5 +93,7 @@ private bool ContainsID(long id)
 		/// A one-to-one dictionary of export id's
 		/// </summary>
 		protected readonly Dictionary<AssetInfo, long> m_exportIDs = new();
```

Comment:
This can be made private and changed to `Dictionary<IUnityObjectBase, long>`. `m_assets` can be removed.